File: trunk/quests/citymist/a_human_skeleton.pl
# Zone: City Of Mist
# Short name: citymist
#
# NPC Name: a_human_skeleton
# NPC ID: 90167, 90170
#

# Quest: Brain Bite (Concussion)
# NPC's Involved: Despondo (Firiona Vie), a_human_skeleton (x3, Trakanon's Teeth, Karnor's Castle, City of Mist)

sub EVENT_SAY {
  if ($text=~/Hail/i) {
    quest::say("Save my.. Soul.. Find.. Wizard.. Desp..");
  }
}

sub EVENT_ITEM {

# Hand in Mardon's Bottle
  if (plugin::check_handin(\%itemcount, 12963 => 1)) {
    quest::summonitem(12966); # Bottle of swirling smoke (Mardon's soul)
    quest::depop();
  }
  plugin::return_items(\%itemcount);
}

File: citymist/a_human_skeleton.lua
-- Quest: Brain Bite (Concussion)
-- NPC's Involved: Despondo (Firiona Vie), a_human_skeleton (x3, Trakanon's Teeth, Karnor's Castle, City of Mist)
function event_say(e)
	if(e.message:findi("Hail")) then
		e.self:Say("Save my.. Soul.. Find.. Wizard.. Desp..");
	end
end

function event_trade(e)
	local item_lib = require("items");
	
	if(item_lib.check_turn_in(e.trade, {item1 = 12963})) then -- Hand in Mardon's Bottle
		e.other:QuestReward(e.self,{itemid = 12966}); -- Bottle of swirling smoke (Mardon's soul)
		eq.depop_with_timer();
	end
	item_lib.return_items(e.self, e.other, e.trade)
end
